observation/bf_phaseup.py

```diff
@@ -150,7 +150,9 @@ def calculate_corrections(G_gains, B_gains, delays, cal_channel_freqs,
         target.add_tags('bfcal single_accumulation')
         user_logger.info("Target to be observed: %s", target.description)
         session.capture_init()
-        session.cbf.correlator.req.capture_start()
+        user_logger.info("Only calling capture_start on correlator streams")
+        for correlator in session.cbf.correlators:
```

Comment:
Since starting data capture for all correlators would be the normal thing to do, should we rather have the loop in session?